src/main/pages/workspace-admin/stacks.adoc

```diff
@@ -71,13 +71,16 @@ image::workspaces/stack_permissions.png[]
 }
 ----
 
-If you get 204, all users in the system will be able to see this stack. If you want to share a stack with a particular user, get its ID and use it instead of `*` in JSON.
+If you get 204, all the users in the system see the stack. If you want to share a stack with a particular user, get the user's ID and use it instead of `*` in the JSON.
 
-Similarly, an admin can remove permissions from stacks or remove out of the box stacks to replace them with custom ones. Stacks can be created wither in User Dashboard or using any REST client, for instance Swagger (`$CHE_HOST:$CHE_PORT/swagger`) bundled with Che.
+The administrator can remove pre-configured stacks and replace them with custom stacks. The administrator can also remove permissions from stacks.  You can create stacks either in the user dashboard or by using any REST client. You can use Swagger (`$CHE_HOST:$CHE_PORT/swagger`) to bundle with Che.
 
-[id="stacks-loading"]
-== Stacks Loading
+[id="loading-stacks"]
+== Loading stacks
 
-Stacks are loaded from a JSON file that is packaged into resources of a special component deployed with a workspace master. This JSON isn’t exposed to users and stack management is performed in User Dashboard (that uses REST API).
+Stacks are loaded from a JSON file that is packaged into resources of a special component that is deployed with the workspace master. This JSON is not exposed to users.   You can perform stack management using REST APIs in the `User Dashboard`.
+
+When a user first starts Che, stacks are loaded from a JSON file only when the database is initialized. This is the default policy that can be changed.   To keep getting stack updates with the new Che stacks, set `CHE_PREDEFINED_STACKS_RELOADONSTART=true` in `che.env`. When set to `true`, `stacks.json` is used to update Che database each time the Che server starts. This means Che gets all the stacks in `stacks.json` and uploads the stacks to a database. This allows you to keep existing custom stacks and get stack updates from new Che releases. New and edited stacks that have fixes in the stack definition are merged in with the other stacks. 
+
+Name conflicts are possible. For example, a name conflicts happens if a new Che version provides a new stack with the name "My Cool Stack" and a stack with this name already exists in the database.  The new stack with the name "My Cool Stack" is not saved to the database because of the name conflict.
```

Comment:
* -> a name conflict
* replace the long example with "A name conflict happens when a new Che version provides a stack with a name that already exists in the database."